articles/ai/TOC.yml

```
@@ -0,0 +1,172 @@
+items:
+- name: Develop AI apps using Azure AI services
+  href: index.yml
+  items:
+    - name: Overview
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)